.github/workflows/ci.yml

```
@@ -91,6 +91,9 @@ jobs:
           podman manifest push $IMAGE_NAME:dev.latest docker://quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:$COMMIT_TAG
           podman manifest push $IMAGE_NAME:dev.latest docker://quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:dev.latest
 
+      - name: Run scorecard tests
+        run: make scorecard
```

A strict check of suggestions could help to not ignore new ones:
```suggestion
        run: make scorecard | tee scorecard.log && ! grep 'Suggestions' scorecard.log
```